homeassistant/components/sensor/zabbix_triggers.py

```diff
@@ -0,0 +1,154 @@
+"""
+Support for Zabbix Sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.zabbix/
+"""
+import logging
+from datetime import datetime
+
+import voluptuous as vol
+
+from homeassistant.helpers.entity import Entity
+import homeassistant.components.zabbix as zabbix
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['zabbix']
+
+_CONF_TYPE = "type"
+_CONF_HOSTIDS = "hostids"
+_CONF_INDIVIDUAL = "individual"
+_CONF_NAME = "name"
+
+_ZABBIX_ID_LIST_SCHEMA = vol.Schema([int])
+
+SCAN_INTERVAL = 30
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(_CONF_HOSTIDS, default=[]): _ZABBIX_ID_LIST_SCHEMA,
+    vol.Optional(_CONF_INDIVIDUAL, default=False): cv.boolean(True),
+    vol.Optional(_CONF_NAME, default=None): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Zabbix sensor platform."""
+    sensors = []
+
+    _LOGGER.info("Connected to Zabbix API Version %s",
+                 zabbix.ZAPI.api_version())
+
+    hostids = config.get(_CONF_HOSTIDS)
+    individual = config.get(_CONF_INDIVIDUAL)
+    name = config.get(_CONF_NAME)
+
+    if individual:
+        # Individual sensor per host
+        if not hostids:
+            # We need hostids
+            _LOGGER.critical("If using 'individual', must specify hostids")
+            return False
+
+        for hostid in hostids:
+            _LOGGER.info("Creating Zabbix Sensor: " + str(hostid))
```

Comment:
This seems very verbose to me. Especially since all state changes will also be logged